### PR TITLE
Add DPMJET

### PR DESCRIPTION
**Pull request by qgp**
- add package DPMJET and use it for AliRoot

### CODE REVIEW

**Comment by dberzano** [comment]
For reference, here is the [Jenkins build](https://alijenkins.cern.ch/job/build-any-ib/6355/).

Test it on CVMFS:

```
/cvmfs/alice-nightlies.cern.ch/bin/alienv enter AliPhysics/v5-08-23-01-rc3-q
```

**Comment by dberzano** [comment]
There is an error:

```
ROOT/v5-34-30-alice7-2(85):ERROR:105: Unable to locate a modulefile for 'DPMJET_ROOT/v3.0.5-1'
```

File: aliroot.sh [hunk 1]
```diff
@@ -74,7 +76,7 @@ proc ModulesHelp { } {
 set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
 module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
 # Dependencies
-module load BASE/1.0 ${ROOT_VERSION:+ROOT/$ROOT_VERSION-$ROOT_REVISION} ${FASTJET_VERSION:+fastjet/$FASTJET_VERSION-$FASTJET_REVISION} ${GEANT3_VERSION:+GEANT3/$GEANT3_VERSION-$GEANT3_REVISION} ${GEANT4_VMC_VERSION:+GEANT4_VMC/$GEANT4_VMC_VERSION-$GEANT4_VMC_REVISION} ${VC_VERSION:+Vc/$VC_VERSION-$VC_REVISION}
+module load BASE/1.0 ${ROOT_VERSION:+ROOT/$ROOT_VERSION-$ROOT_REVISION} ${DPMJET_VERSION:+DPMJET_ROOT/$DPMJET_VERSION-$DPMJET_REVISION} ${FASTJET_VERSION:+fastjet/$FASTJET_VERSION-$FASTJET_REVISION} ${GEANT3_VERSION:+GEANT3/$GEANT3_VERSION-$GEANT3_REVISION} ${GEANT4_VMC_VERSION:+GEANT4_VMC/$GEANT4_VMC_VERSION-$GEANT4_VMC_REVISION} ${VC_VERSION:+Vc/$VC_VERSION-$VC_REVISION}
```
Comment (dberzano) [inline aliroot.sh]: Here you have a mistake, it should be:

```
${DPMJET_VERSION:+DPMJET/$DPMJET_VERSION-$DPMJET_REVISION}
```

Comment (qgp) [inline aliroot.sh]: Thanks, for spotting. PR updated accordingly.

**Comment by dberzano** [comment]
OK thanks for the fix! Waiting for a positive feedback from the tests before merging to next.

**Pull request closed without merging**